packages/argo-checkout/src/components/Banner/Banner.ts

```diff
@@ -0,0 +1,43 @@
+import {createRemoteComponent} from '@remote-ui/core';
+
+/**
+ * Success:
+ * Use rarely, only if you need additional visual confirmation that a
+ * non-standard action has been completed successfully, for example adding an
+ * item to an order as an upsell.
+ *
+ * Warning:
+ * Use to display information that needs attention or that customers need to
+ * take action on. Seeing these banners can be stressful for customers so be
+ * cautious about using them. Does not block progress to next step.
+ *
+ * Critical:
+ * Use to communicate problems that have to be resolved immediately for
+ * customers to complete a task. For example, using a different payment method
+ * if card details couldn't be processed. Seeing these banners can be stressful
+ * for customers so be cautious about using them.
+ */
+type Status = 'info' | 'success' | 'warning' | 'critical';
```

Comment:
We're missing the `info` description.

Reply:
yeah, I copied this from checkout-web-ui; I'm guessing it wasn't there because it's the default? I'll see if I can cobble something together.